### PR TITLE
Add Last Updated field to Traffic Portal

### PR DESCRIPTION
**Pull request by shamrickus**
This PR adds the Last Updated field to every DS form in Traffic Portal (tpv2 already has them).
<hr/>
## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
Verify the last updated field is present on each type of Delivery Service and is correct in Traffic Portal.

## PR submission checklist
- [x] This PR has tests 
- [x] This PR has documentation 
- [x] This PR has a CHANGELOG.md entry 
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY** (see [the Apache Software Foundation's security guidelines](https://apache.org/security) for details)

### CODE REVIEW

**Comment by rimashah25** [comment]
I see the `Last Updated` field in the form correctly
image
but in the DS table it shows as Invalid Date
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - *Traffic Monitor* Added logging for `ipv4Availability` and `ipv6Availability` in TM.
 - *Traffic Ops* Added API version 5.0
 - *Traffic Ops* Added the `ASN` field in TO Server struct, which provides the ability to query servers by `ASN`.
+- *Traffic Portal* Added the `lastUpdated` field to all delivery service forms.
```
Comment (mitchell852) [inline CHANGELOG.md:11]: @shamrickus - would you mind adding a link to the issue or PR?

Comment (mitchell852) [reply CHANGELOG.md:11]: also, i can merge this if you fix the conflicts in this file.

**Review by rimashah25 — APPROVED** [review]
Code LGTM. On local I still see the invalid date (even for master branch). So, it seems like a local issue and not related to this PR. 
Need to resolve conflicts before code can be merged.

File: CHANGELOG.md (hunk 1 above)
Comment (mitchell852) [reply CHANGELOG.md:11]: nm, looks like @rimashah25 found a bug

File: CHANGELOG.md [hunk 2]
```diff
@@ -47,7 +48,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - [Traffic Portal] Added the ability for users to view Delivery Service Requests corresponding to individual Delivery Services in TP.
 - [Traffic Ops] Added support for backend configurations so that Traffic Ops can act as a reverse proxy for these services [#6754](https://github.com/apache/trafficcontrol/pull/6754).
 - Added functionality for CDN locks, so that they can be shared amongst a list of specified usernames.
-- [Traffic Ops | Traffic Go Clients | T3C] Add additional timestamp fields to server for queuing and dequeueing config and revalidate updates.
+- [Traffic) Ops | Traffic Go Clients | T3C] Add additional timestamp fields to server for queuing and dequeueing config and revalidate updates.
```
Comment (mitchell852) [inline CHANGELOG.md:51]: looks like a misplaced paren

**Review by ocket8888 — APPROVED** [review]

**Pull request merged**